samples/nrf_desktop/src/modules/hid_forward.c

```diff
@@ -93,19 +93,20 @@ static u8_t hidc_read(struct bt_gatt_hids_c *hids_c,
 
 static void hidc_ready(struct bt_gatt_hids_c *hids_c)
 {
-	for (size_t i = 0; i < hids_c->rep_cnt; i++) {
-		struct bt_gatt_hids_c_rep_info *rep = hids_c->rep_info[i];
-
-		if (rep->ref.type == BT_GATT_HIDS_C_REPORT_TYPE_INPUT) {
-			int err = bt_gatt_hids_c_rep_subscribe(&hidc,
-							       hidc.rep_info[i],
+	struct bt_gatt_hids_c_rep_info *rep = NULL;
```

Comment:
double assignment to `rep`
(`rep` will be assigned in line 97)

Reply:
Yes - but it is used as a parameter for the bt_gatt_hids_c_rep_next too. And giving NULL here we are sure that first report is returned on first iteration.